campaigns.json

```diff
@@ -781,7 +785,44 @@
     }, 
     "secondary_AAA": false, 
     "tune": true
-  }, 
+  },
+  "Phase2Spring21DRMiniAOD":{
+    "fractionpass": 0.95,
+    "go": true,
+    "lumisize": -1,
+    "maxcopies": 1,
+    "resize": "auto",
+    "secondaries": {
+           "/MinBias_TuneCP5_14TeV-pythia8/Run3Winter20GS-110X_mcRun3_2021_realistic_v6-v1/GEN-SIM": {
+                "SiteWhitelist": [
+                         "T1_ES_PIC_Disk",
+                         "T1_US_FNAL_Disk"
+                                 ]
+                                     }
+                              },
+    "secondary_AAA": false  
+  },
+  "Phase2D81Spring21GS":{
+      "fractionpass": 0.95,
+      "go": true,
```

Comment:
Pilots are not finished, yet. Why go True? This comment applies to the other Phase2 campaigns as well.

Reply:
You are correct this I greatly appreciate. I copied and pasted but found another issue and got distracted.

Reply:
@scarletnorberg  this is not changed in your last commits. Have you missed it?